### PR TITLE
Fix vcfparser

### PR DESCRIPTION
**Pull request by yunfeiguo**
- [ ] [RES-XXXX](https://binatechnologies.atlassian.net/browse/RES-XXXX)
- [ ] Blocking PRs: ADD HERE IF ANY
* People
  - [ ] Reviewers (>=1): ADD HERE
  - [ ] Merger (>=1): ADD HERE
  - [ ] FYI: ADD HERE IF ANY
- [ ] Release note (if needed)
- [ ] No code copied from outside Bina 
- [ ] Tests (Unit, Workflow, ITL & Gherkin) are passing
- [ ] PR labels, milestones & assignees

# Summary

check for illegal char in ATL

### CODE REVIEW

File: src/main/java/com/bina/varsim/util/VCFparser.java [hunk 1]
```diff
@@ -347,7 +347,13 @@ else if (index == 9) { // Output format
                 }
             }
         }
-        Alt[] alts = string2Alt(ALT);
+        Alt[] alts = null;
+        try {
+            alts = string2Alt(ALT);
+        } catch (IllegalArgumentException e) {
+            log.warn(e.getMessage());
```
Comment (marghoob) [inline src/main/java/com/bina/varsim/util/VCFparser.java]: Would be good to print out the offending VCF line

Comment (yunfeiguo) [reply src/main/java/com/bina/varsim/util/VCFparser.java]: good point, just did. Thanks.

File: src/main/java/com/bina/varsim/util/VCFparser.java [hunk 2]
```diff
@@ -347,7 +347,14 @@ else if (index == 9) { // Output format
...
+            log.warn(line);
```
Comment (marghoob) [inline src/main/java/com/bina/varsim/util/VCFparser.java]: Would be better to make it more informative, so append some message here.

Comment (yunfeiguo) [reply src/main/java/com/bina/varsim/util/VCFparser.java]: done. Thanks!

**Comment by marghoob** [comment]
👍

**Comment by yunfeiguo** [comment]
Thanks. Merging...

**Pull request merged**